api/hub/src/datasources/moderation-decision.ts

```diff
@@ -203,7 +204,8 @@ class ModerationDecisionAPI extends DataSource {
       const decision = await this.getFinalDecision(result.contentID);
       // load moderator info
       const profileAPI = new ProfileAPI({ dbID: this.dbID, collection: 'Profiles' });
-      const moderator = await profileAPI.getProfile(decision.moderator);
+      const moderator = decision.moderator.startsWith('0x') ? 
```

Comment:
is this still valid if moderators have been migrated to pubKeys?

Reply:
You are right, this can be removed now.